src/ontology/foodon-idranges.owl

```diff
@@ -94,6 +94,22 @@ Datatype: idrange:10
     EquivalentTo: 
         xsd:integer[> 2230000 , <= 2399999]
 
+Datatype: idrange:13
+
+    Annotations: 
+        allocatedto: "Fungal template (generated)"
+    
+    EquivalentTo: 
+        xsd:integer[> 2400000 , <= 2599999]
+
+Datatype: idrange:14
```

Comment:
I think we could just have this "Fungal parts (manual curation)" range be part of the "Animal and Seafood parts (manual curation)" range, but lets rename this to "Manually curated organism parts" range - Datatype: idrange:11
    
    EquivalentTo: 
        xsd:integer[> 2010000 , <= 2019999]